tests/Tests.AzureAppConfiguration/RefreshTests.cs

```diff
@@ -101,10 +121,43 @@ public void RefreshTests_RefreshIsSkippedIfCacheIsNotExpired()
                 .AddAzureAppConfiguration(options =>
                 {
                     options.Client = mockClient.Object;
-                    options.Select("TestKey*");
+                    // NOTE: This select clause is redundant, we are loading all settings by default in GetMockConfigurationClient()
+                    options.Select("TestKey*"); 
```

Comment:
We could just remove these two lines, since these are not relevant to the scenario we're testing.